src/product-service-java/src/main/java/com/microsoft/azure/productservice/config/WebMvcConfig.java

```diff
@@ -0,0 +1,29 @@
+package com.microsoft.azure.productservice.config;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.stereotype.Component;
+import org.springframework.web.servlet.HandlerInterceptor;
+import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+
+@Configuration
+public class WebMvcConfig implements WebMvcConfigurer {
```

Comment:
Instead of adding a custom header here, proper versioning should be used, i.e. https://www.springboottutorial.com/spring-boot-versioning-for-rest-services#custom-headers-versioning

Reply:
@guybartal  please take a look at this PR